core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1632,6 +1632,8 @@ private CarbonCommonConstants() {
 
   public static final String S3A_PREFIX = "s3a://";
 
+  public static final String LOCAL_FS_URI = "file:///";
```

Comment:
keep it as `LOCAL_FS_PREFIX` to be inline with other file system notation.

Reply:
If you see the FSConstants class in hdfs, the variable name is the same, so I followed the same here too. I think its fine.

Reply:
ok. fine